### PR TITLE
Fix issue #1309

### PR DESCRIPTION
**Pull request by mafeijam**
this is to fix issue #1309 when executing command from the web instead of CLI

### CODE REVIEW

**Comment by freekmurze** [comment]
Could we improve on this by only doing the `defined` check?

**Comment by mafeijam** [comment]
just borrow the idea from Laravel Application class - `runningInConsole` method

maybe we can check for `in_array(php_sapi_name(), ['cli', 'phpdbg'])` to ensure the command is running in console instead of dong `defined` check.

**Comment by karticbedi** [comment]
@freekmurze any ETA when you will consider this commit ? we are waiting to push our code to production because of this error.

**Comment by freekmurze** [comment]
Checking on `in_array(php_sapi_name(), ['cli', 'phpdbg'])` and `defined('SIGINT')` seems like the way to go here.

File: src/Commands/BaseCommand.php [hunk 1]
```diff
@@ -26,4 +26,9 @@ public function run(InputInterface $input, OutputInterface $output): int
 
         return parent::run($input, $output);
     }
+    
+    protected functiin runningInConsole(): bool
```
Comment (cretueusebiu) [inline src/Commands/BaseCommand.php:30]: Typo here `functiin` -> `function`

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**